platform-tests/src/test/java/org/junit/platform/suite/commons/SuiteLauncherDiscoveryRequestBuilderTests.java

```diff
@@ -214,6 +217,88 @@ class Suite {
 		assertEquals(TestCase.class, exactlyOne(selectors).getJavaClass());
 	}
 
+	@Test
+	void selectMethodsOnlyByName() {
+		class TestClass {
+			public void testMethod() {
+			}
+
+			public void anotherTestMethod() {
+			}
+		}
+		@SelectMethods({ @SelectMethod(name = "testMethod"), @SelectMethod(name = "anotherTestMethod") })
+		class Suite {
```

Comment:
You can repeat repeatable annotations directly without the outer `@SelectMethods`.